lerobot/common/datasets/video_utils.py

```diff
@@ -20,6 +20,7 @@
 import warnings
 from collections import OrderedDict
 from dataclasses import dataclass, field
+from  lerobot.common.constants import GPU_ENCODING
```

Comment:
style nit - make sure you're using pre-commit

```suggestion
from lerobot.common.constants import GPU_ENCODING
```